index.html

```diff
@@ -1102,7 +1102,7 @@ <h3>Identifiers</h3>
 <a>property</a> is intended to unambiguously refer to an object such as a person,
 product, or organization. This allows for the expression of statements about
 specific things in the verifiable credential. If the <code>id</code>
-<a>property</a> is present:
+<a>property</a> MAY be present, if so:
         </p>
```

Comment:
``` The <code>id</code>
<a>property</a> MAY be present; if so:```
* remove leading "If"
* change comma to semicolon

Reply:
resolved in [89d1e1f](https://github.com/w3c/vc-data-model/pull/616/commits/89d1e1f364ae3fd2a15328247fb8943d18b98825)